src/Diagnostics.ModelsAndUtils/Models/DataTableResponseObject.cs

```diff
@@ -17,7 +17,7 @@ public class DataTableResponseObject
 
         public IEnumerable<DataTableResponseColumn> Columns { get; set; }
 
-        public string[][] Rows { get; set; }
+        public dynamic[,] Rows { get; set; }
```

Comment:
>dynamic[,] Rows [](start = 15, length = 15)

Just to make sure I'm understanding this correctly, did the deserialization error occur because arrays or objects from kusto response string could not be deserialized to strings?

Reply:
Correct. There was something there in the value that looked like 

`['value','value',['array','from','makeset']]`

and it choked on the inner array when it tried to read it as string.